### ISSUE TITLE
Accessibility: Individual points in the graph do not announce their appropriate role (link) and a descriptive name

### ISSUE THREAD

**Issue by rezinak** [highcharts/highcharts#18523]
#### Expected behaviour
The screen reader announces an appropriate role (link) for each individual point in the graph.

Since the individual points in the graph lead the user to another URL, you should ensure the individual points have descriptive names that mention their purpose. For example, the accessible name for the individual section can be "December 2010 JAWS 69.6% visit JAWS website"

Ensure custom controls provide proper textual name, role, and state information.

Note: The best way to do this is to use native controls, as they come with this information built in. In this case, an <a> element with a non-null HREF would be most appropriate for these controls. An HREF is required because <a> elements without HREF attributes are not always considered links by assistive technology. If necessary, developers can set the HREF to "#" or "javascript:void(0);".

If using <a> elements is not possible, the links must be given role="link". If the links have insufficient internal text, developers must add an aria-label attribute or an aria-labelledby attribute to provide a name.

Relevant WCAG criteria: [Non-text content](https://www.w3.org/TR/WCAG21/#non-text-content), [name-role-value](https://www.w3.org/TR/WCAG21/#name-role-value).

#### Actual behaviour
Each individual point in the graph (December 2010 JAWS 69.6%) behaves as a link. However, the individual points do not announce their appropriate role (link) and a descriptive name (December 2010 JAWS 69.6% visit JAWS website).

Screen reader users will be unable to determine that these controls navigate to new pages.

#### Live demo with steps to reproduce
https://www.idc.com/QA_test_section/accessibility-remediation/highcharts-demo1 

#### Product version
Highcharts JS v10.3.3

#### Affected browser(s)
Chrome and Mozilla, haven't looked into the others

**Comment by oysteinmoseng** [comment]
Hi, we have some user research testing and some improvements to this in the pipeline. Marking as a bug for now.
